diff --git a/op-node/rollup/derive/system_config.go b/op-node/rollup/derive/system_config.go
index 72c4e713c..c8141d514 100644
--- a/op-node/rollup/derive/system_config.go
+++ b/op-node/rollup/derive/system_config.go
@@ -21,7 +21,8 @@ var (
 	SystemConfigUpdateFeeScalars        = common.Hash{31: 1}
 	SystemConfigUpdateGasLimit          = common.Hash{31: 2}
 	SystemConfigUpdateUnsafeBlockSigner = common.Hash{31: 3}
-	SystemConfigUpdateEIP1559Params     = common.Hash{31: 4}
+	SystemConfigUpdateGasConfigEcotone  = common.Hash{31: 4}
+	SystemConfigUpdateEIP1559Params     = common.Hash{31: 5}
 )

 var (
@@ -160,6 +161,9 @@ func ProcessSystemConfigUpdateLogEvent(destSysCfg *eth.SystemConfig, ev *types.L
 	case SystemConfigUpdateUnsafeBlockSigner:
 		// Ignored in derivation. This configurable applies to runtime configuration outside of the derivation.
 		return nil
+	case SystemConfigUpdateGasConfigEcotone:
+		// HOTFIX: Ignore legacy ConfigUpdate event GAS_CONFIG_ECOTONE that shouldn't be used anymore
+		return nil
 	default:
 		return fmt.Errorf("unrecognized L1 sysCfg update type: %s", updateType)
 	}
